app/src/main/java/com/youssef/fixit/Auth/LoginFragment/LoginFragment.java

```diff
@@ -159,12 +158,17 @@ public void OnForgotPassword() {
         Toast.makeText(getContext(), "Mail is Sent", Toast.LENGTH_SHORT).show();
     }
 
+    @Override
+    public void ShowLoading() {
+        dialog.show();
+    }
+
     void SaveDataInShared(String token, String role, int my_id) {
```

Comment:
`saveDataInShared` can be private, right?

Reply:
right 
i think default access modifier is protected right?